pkg/collector/corechecks/cluster/orchestrator/processors/processor.go

```diff
@@ -138,23 +148,40 @@ func (p *Processor) Process(ctx *ProcessorContext, list interface{}) (messages [
 		}
 
 		// Execute code after marshalling.
-		if skip := p.h.AfterMarshalling(ctx, resource, resourceModel, yaml); skip {
+		if skip := p.h.AfterMarshalling(ctx, resource, resourceMetadataModel, yaml); skip {
 			continue
 		}
 
-		resourceModels = append(resourceModels, resourceModel)
+		resourceMetadataModels = append(resourceMetadataModels, resourceMetadataModel)
+
+		// Add resource manifest
+		resourceManifestModels = append(resourceManifestModels, &model.Manifest{
+			Type:            int32(ctx.NodeType),
```

Comment:
Can be in a follow up, be lets have the discussion: Another big wonder, could we auto sync the manifest type with the actual type? I know it conflicts with the CollectorType but we might be able to infer the type another way? This way we don't need to maintain 2 lists which are basically the same

Reply:
I decide to use the node type that we have already defined instead of using collector type here  https://github.com/DataDog/datadog-agent/blob/8812f91bda38675426b8c3e2ca27bdf1c1692c3b/pkg/orchestrator/util.go#L20-L26

Before I converted the `NodeType` to `CollectorType `  https://github.com/DataDog/datadog-agent/blob/45bf8f76a82f435e337128969d832d40b36889ee/pkg/collector/corechecks/cluster/orchestrator/processors/processor.go#L159 .

They are exactly the same "type" (same int32 value ) as we used in the backend.